.github/workflows/verify_published.yml

```diff
@@ -38,5 +38,5 @@ jobs:
           
           if [ "${{ matrix.extras }}" = "[all]" ]; then
             # Verify some of the optional dependencies are available
-            python -c "import tifffile; from pylibCZIrw import czi;"
+            python -c "import tifffile; import imagecodecs; import pandas;"
```

Comment:
- Replace CZI with some others libs from the extras:
<img width="379" alt="Screenshot 2024-11-26 at 10 27 05" src="https://github.com/user-attachments/assets/faae6364-014d-407a-a70d-85a9f94def5e">